### PR TITLE
add landing page

### PR DESCRIPTION
**Pull request by andrewdamelio**
#3916 
image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 😍

 This link will always point to this PR's latest changes: https://peek.run/p37hvx3

File: packages/augur-ui/src/modules/markets-list/selectors/markets-list.ts [hunk 1]
```diff
@@ -66,13 +70,50 @@ export const selectAllOtherCategories = createSelector(
...
-        .sort((a, b) => a.count > b.count ? -1 : 1);
+        .sort((a, b) => (a.count > b.count ? -1 : 1));
 
       return allOtherCategories;
     }
     return null;
   }
 );
+
+export const selectMarketStats = createSelector(
```
Comment (bthaile) [inline packages/augur-ui/src/modules/markets-list/selectors/markets-list.ts:83]: new getter on this PR https://github.com/AugurProject/augur/pull/3971

to get category stats, meant to be used for market creation templates, we can prob use it for market stats as well. lets chat

**Comment by bthaile** [comment]
What is shown to the user while all the data is being sync'd and gathered. looks like they will just see a blank page.

**Comment by bthaile** [comment]
Getting this on your branch. I cleared application data and reloaded. it happens after logging in.

image

actually I see it on master.

File: packages/augur-ui/src/modules/markets-list/selectors/markets-list.ts (hunk 1 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/markets-list/selectors/markets-list.ts:83]: excellent! i'll wire that up in another PR.

**Comment by andrewdamelio** [comment]
> What is shown to the user while all the data is being sync'd and gathered. looks like they will just see a blank page.

@bthaile are you seeing a blank page on featurepeek? It should show loading state like the below
image


> Getting this on your branch. I cleared application data and reloaded. it happens after logging in.
> 
> image
> 
> actually I see it on master.

Also i don't see the above error - how are you triggering it?

**Review by bthaile — APPROVED** [review]
lgtm even though it's not wired up the the site.

**Pull request merged**